### ISSUE TITLE
jfxtras-styles does not build in Java 10 on macOS High Sierra (10.13.6)

### ISSUE THREAD

**Issue by wbeebe** [JFXtras/jfxtras-styles#11]
When attempting to build the following types of errors are generated:

> Task :jmetro:compileJava
/Users/wbeebe/Git/jfxtras-styles/jmetro/src/main/java/impl/jfxtras/styles/jmetro8/TextFieldWithButtonSkin.java:31: error: cannot find symbol
import com.sun.javafx.scene.control.skin.TextFieldSkin;

Investigation shows that according to JEP 253 that:

The intention is to move many JavaFX control skins into the appropriate public package, most probably javafx.scene.control.skin. There is no intent to also move the related behavior classes.

(see http://openjdk.java.net/jeps/253)

It is my intention to
1. Fork jfxtras-styles
2. Attempt to find and fix these errors
3. Propose the fixes back to the project via a pull request.

I make no guarantees this will be straightforward, or that I will succeed. But someone needs to try.

**Comment by dukke** [comment]
Apologies for the late reply. Unfortunately due to a configuration issue I wasn't getting any notifications when new issues were entered in this project.

**Comment by dukke** [comment]
This project is currently focused on Java 8, since according to statistics that seems to be, currently, the most used version of Java. Also, since I don't have that much time I decided to currently only focus on one version of Java.
In the future I'll probably update the Java version compatibility, but to the most recent one. I'll probably won't care about intermediate Java versions.

Having said that, I did try to make JMetro easy to port to more recent versions of Java by not using internal API and API that would be replaced or removed in Java 9+.

Skins are indeed public since Java 9 so it's probably fairly easy to port. The hard work might be in "modularizing" the project.

 A PR would be very welcome! A new branch for Java 9 would probably be created.
